### PR TITLE
[MX-496] Use palette's EntityHeader on FavoriteArtists list

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Enhancement**

This PR resolves **[MX-496]**

### Description

Use palette's `EntityHeader` on FavoriteArtists list  instead of `SavedItemRow`

### Screenshots
Screen_Recording_2020-08-13_at_10 48 18

#trivial

[MX-496]: https://artsyproduct.atlassian.net/browse/MX-496

### CODE REVIEW

File: src/lib/Scenes/Favorites/FavoriteArtists.tsx [hunk 1]
```diff
@@ -88,7 +86,7 @@ class Artists extends React.Component<Props, State> {
         onEndReached={this.loadMore}
         onEndReachedThreshold={0.2}
         refreshControl={<RefreshControl refreshing={this.state.refreshingFromPull} onRefresh={this.handleRefresh} />}
-        style={{ paddingTop: 15, paddingHorizontal: 0 }}
+        style={{ paddingTop: 15, paddingHorizontal: 15 }}
         ItemSeparatorComponent={() => <Spacer mb="5px" />}
```
Comment (ds300) [inline src/lib/Scenes/Favorites/FavoriteArtists.tsx:90]: can you bump this up to 15px?

Comment (MounirDhahri) [reply src/lib/Scenes/Favorites/FavoriteArtists.tsx:90]: yes of course 🙏

**Comment by ds300** [comment]
Looking at this again i noticed that the padding around the list is wrong (visually, there should be 20px padding) and also that we lost the touch interaction for the list rows.

We should use the `add-background` interaction outlined here https://www.notion.so/artsy/Motion-Guidelines-11000a5ea9764bf182cc0732f5cebb17#1df138bbe6c442e5a6c92722e13e2b47

**Comment by MounirDhahri** [comment]
done, please check the updated screenshots

**Comment by ds300** [comment]
Looks good! The `add-background` effect should stretch the full width of the screen though, and there should be 5px of vertical padding on the touchable.

**Comment by MounirDhahri** [comment]
No worries 😋

**Pull request merged**